src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/reference/inputs/EceWorkExperienceReferenceDetails.vue

```diff
@@ -0,0 +1,227 @@
+        </v-col>
+      </v-row>
+      <v-row>
+        <v-col cols="12" md="8" lg="6" xl="4">
+          <p>What is your relationship to the applicant?</p>
+          <v-radio-group
+            hide-details="auto"
+            :rules="[Rules.requiredRadio('Select an option')]"
+            @update:model-value="updateField('referenceRelationship', $event)"
+          >
+            <v-radio
+              v-for="(workReferenceRelationship, index) in workReferenceRelationshipRadio"
+              :key="index"
+              :label="workReferenceRelationship.label"
+              :value="workReferenceRelationship.value"
+            ></v-radio>
+          </v-radio-group>
+        </v-col>
+      </v-row>
+      <v-row v-if="modelValue.referenceRelationship === 'Other'" class="mt-5">
+        <v-col cols="12" md="8" lg="6" xl="4">
+          <v-text-field
+            label="Specify Your relationship"
+            variant="outlined"
+            color="primary"
+            :rules="[Rules.required('Specify your relationship to the applicant. Your response should not exceed 100 characters')]"
+            hide-details="auto"
+            maxlength="100"
+            @update:model-value="updateField('referenceRelationshipOther', $event)"
+          ></v-text-field>
+        </v-col>
+      </v-row>
+    </v-col>
+  </v-row>
+</template>
+
+<script lang="ts">
+import { DateTime } from "luxon";
+import { defineComponent } from "vue";
+
+import { useWizardStore } from "@/store/wizard";
+import type { Components } from "@/types/openapi";
+import { childrenProgramTypeDropdown } from "@/utils/constant";
+import { workHoursTypeRadio } from "@/utils/constant";
+import { childcareAgeRangesCheckBox } from "@/utils/constant";
+import { workReferenceRelationshipRadio } from "@/utils/constant";
+import { formatDate } from "@/utils/format";
+import * as Rules from "@/utils/formRules";
+
+export default defineComponent({
+  name: "EceWorkExperienceReferenceDetails",
+  props: {
+    modelValue: {
+      type: Object as () => Components.Schemas.WorkExperienceReferenceDetails,
+      required: true,
+    },
+  },
+  emits: {
+    "update:model-value": (_workExReferenceData: Components.Schemas.WorkExperienceReferenceDetails) => true,
+  },
+  setup: () => {
+    const wizardStore = useWizardStore();
+
+    return { wizardStore, childrenProgramTypeDropdown, workHoursTypeRadio, childcareAgeRangesCheckBox, workReferenceRelationshipRadio };
+  },
+  data() {
+    return {
+      Rules,
+    };
+  },
+
+  computed: {
+    today() {
+      return formatDate(DateTime.now().toString());
+    },
+  },
+  methods: {
+    updateField(fieldName: keyof Components.Schemas.WorkExperienceReferenceDetails, value: any) {
+      this.$emit("update:model-value", {
+        ...this.modelValue,
+        [fieldName]: value,
+      });
+    },
+    referenceRelationshipChanged(value: Components.Schemas.ReferenceRelationship) {
```

Comment:
oh you can refer to this one :) to update the wizard when we change the relationship type to other.